R/sample.R

```diff
@@ -242,6 +269,7 @@ append_chains <- function(prev, curr) {
 
   samples <- list(pars = pars,
                   density = density,
+                  initial = prev$initial,
```

Comment:
Should we hold a history of the initial points of each "continue" too? Or is the user expected to hold that if they need it?

Reply:
The initial points of continue are different; they are within the chain already (the last point of the previous set) and once the chains are merged the idea is that they are indistinguishable from one chain run end to end.  So the (real) initial is the only additional bit of data we need to retain